src/test/java/org/xmlcml/cml/element/main/CMLDictionaryTest.java

```diff
@@ -0,0 +1,103 @@
+/**
+ *    Copyright 2023 Sivasuriyan
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package org.xmlcml.cml.element.main;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xmlcml.cml.element.CMLDictionary;
+import org.xmlcml.cml.element.CMLEntry;
+
+public class CMLDictionaryTest {
+
+    protected CMLDictionary cmlDictionary = null;
```

Comment:
Depending on how to JUnit tests are run, the individual tests are not necessarily run in the order in which they are given in the source code, particularly, when you run tests in parallel. I have had problems with this in the past. It makes the tests slower, I know, but may I suggest to have a new `CMLDictionary` with each test, and not use global variables? I totally understand you did, but it's more stable without.

Reply:
Done